kvdb-rocksdb/src/lib.rs

```diff
@@ -240,30 +240,49 @@ impl DBAndColumns {
 struct DbStats {
 	reads: AtomicU64,
 	writes: AtomicU64,
+	write_bytes: AtomicU64,
+	read_bytes: AtomicU64,
 	transactions: AtomicU64,
 	started: Mutex<std::time::Instant>,
 }
 
 struct TakenDbStats {
 	reads: u64,
 	writes: u64,
+	write_bytes: u64,
+	read_bytes: u64,
 	transactions: u64,
 	start: std::time::Instant,
 }
 
 impl DbStats {
 	fn new() -> Self {
-		Self { reads: 0.into(), writes: 0.into(), transactions: 0.into(), started: std::time::Instant::now().into(), }
+		Self {
+			reads: 0.into(),
+			read_bytes: 0.into(),
+			writes: 0.into(),
+			write_bytes: 0.into(),
+			transactions: 0.into(),
+			started: std::time::Instant::now().into(),
+		}
 	}
 
 	fn add_reads(&self, val: u64) {
 		self.reads.fetch_add(val, AtomicOrdering::Relaxed);
 	}
 
+	fn add_read_bytes(&self, val: u64) {
+		self.read_bytes.fetch_add(val, AtomicOrdering::Relaxed);
+	}
+
 	fn add_writes(&self, val: u64) {
 		self.writes.fetch_add(val, AtomicOrdering::Relaxed);
 	}
 
+	fn add_write_bytes(&self, val: u64) {
```

Comment:
I suggest we rename this to `tally_bytes_written` to convey that we're keeping a total count. Same for the other `add_*` methods.